woocommerce/woocommerce-ios
# WPAuthenticator - Config flag to enable custom step value for get started screen.

- [x] ⚠️  @selanthiraiyan Release WPAuthenticator pod and update podfile.



Related to: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/724



## Description

To avoid changing the tracks event for WordPress Mobile application, we have introduced a configuration flag in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/724

This PR uses the new configuration flag to get a custom step value for the tracks event from `GetStartedViewController` 

## Testing instructions
- Log out of the app or skip onboarding if needed.
- On the prologue screen, select "Continue with WordPress.com" land in the `GetStartedViewController` (Ref screenshot below)
- Check the Xcode console and ensure that the following event is tracked
```
Tracked unified_login_step, properties: [AnyHashable("source"): "default", AnyHashable("step"): "enter_email_address", AnyHashable("flow"): "wordpress_com"]
```
- Notice that the step value is `enter_email_address` as we have enabled the `useEnterEmailAddressAsStepValueForGetStartedVC` flag in `WordPressAuthenticatorConfiguration`

## Screenshots

<img src="https://user-images.githubusercontent.com/524475/213075906-b9ff45ac-c341-4844-8d82-f37905cd0d69.png" width="320"/>

---
